aws/ecs/cs-on-premises/cluster.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_security_group" "lb" {
+  name        = "cs-load-balancer-security-group"
+  description = "controls access to the ALB"
+  vpc_id      = aws_vpc.vpc.id
+
+  ingress {
+    protocol    = "tcp"
+    from_port   = 80
+    to_port     = 80
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
It's worth mentioning in the README.md that the load balancer configuration creates only a HTTP listener, without HTTPS